media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```
@@ -0,0 +1,128 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+_COMMIT="a97b03fad"
+MY_PV="${PV}-${_COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server-new"
+#URI="https://artifacts.plex.tv/plex-media-server-experimental/"
+SRC_URI="
+	amd64? ( ${URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="system-openssl"
+
+BDEPEND="dev-util/patchelf"
+
+RDEPEND="
+	acct-group/plex
+	acct-user/plex
+	system-openssl? ( dev-libs/openssl )
+	"
+# rdepend not working yet
+#custom-ffmpeg? ( media-video/plex-ffmpeg )
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=( "usr/lib/plexmediaserver/lib/.*"
```

Style: new line before the element of the array, so
```suggestion
QA_MULTILIB_PATHS=(
    "usr/lib/plexmediaserver/lib/.*"
    "usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
    "usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
)
```